src/playercards/AllCardsBag.ttslua

```diff
@@ -1,8 +1,144 @@
 local cardIdIndex = { }
-local classAndLevelIndex = { }
+local otherIndex
```

Comment:
For the same reason as the iconIndex idea above, nesting important subindices like this breaks other functionality.  Additionally, these cards are tracked totally differently (set-style, rather than map style), so it would make things very messy to mix them  (as explained more in the other comment below on the otherIndex).